ocaml/xapi/message_forwarding.ml

```diff
@@ -835,10 +835,10 @@ module Forward = functor(Local: Custom_actions.CUSTOM_ACTIONS) -> struct
 
 		(* Used by VM.start_on, VM.resume_on, VM.migrate to verify a host has enough resource and to
 		   'allocate_vm_to_host' (ie set the 'scheduled_to_be_resident_on' field) *)
-		let reserve_memory_for_vm ~__context ~vm ~snapshot ~host ?host_op f =
+		let reserve_memory_for_vm ~__context ~vm ~snapshot ~host ?host_op ?(do_migrate_check=false) f =
```

Comment:
We are already passing in `host_op` here, which can be ``` `vm_migrate ```. Can we use that instead of adding a new argument to the function?